docs/connectors.md

```diff
@@ -18,12 +23,69 @@ Give the issue a `+1` if you really need that connector. The upvote will help th
 particular connector. If you find that an issue hasn't been created for your data store, please create a new issue in
 the Conduit repo.
 
-## Connectors
+## More about Connectors
+
+### Built-in connectors
+
+To help developers bootstrap pipelines much more quickly,
+Conduit ships with several built-in connectors by default.
+This includes the Postgres, File, Random Data Generator, Kafka and Amazon S3 connectors.
+
+If you are creating and distributing customized, pre-built Conduit binaries yourself,
+you may want to modify the list of built-in connectors, too.
+By modifying the list of built-in connectors,
+your compiled binary will include a set of pre-installed connectors specific to your end users' needs,
+and it can be run by others, as is, without them having to follow any additional installation instructions.
+
+If you want to create your own Conduit binary with a different set of built-in connectors,
+you can build-in an [existing standalone connector](#the-list).
```

Comment:
I wouldn't call it "standalone" in this case. It's just a "connector" until you either build it into the Conduit binary (built-in) or run it in a separate binary (standalone).
```suggestion
you can build-in an [existing connector](#the-list).
```

Reply:
Good call and thank you for the suggestion!